webui/react/src/pages/ProjectDetails.tsx

```diff
@@ -931,7 +931,8 @@ const ProjectDetails: React.FC = () => {
     <Page
       bodyNoPadding
       containerRef={pageRef}
-      docTitle="Project Details"
+      // for docTitle, when id is 1 that means Uncategorized from webui/react/src/routes/routes.ts
```

Comment:
The difference between `Uncategorized` and `Project Detail` are the project `id` in path, so I implemented this way. 
@trentwatt what do you think?
reference https://github.com/determined-ai/determined/blob/master/webui/react/src/routes/routes.ts#L32